proxy/http/HttpBodyFactory.cc

```diff
@@ -655,25 +623,21 @@ HttpBodyFactory::determine_set_by_language(StrList *acpt_language_list, StrList
 HttpBodyTemplate *
 HttpBodyFactory::find_template(const char *set, const char *type, HttpBodySet **body_set_return)
 {
-  RawHashTable_Value v;
-
   Debug("body_factory", "calling find_template(%s,%s)", set, type);
 
   *body_set_return = nullptr;
 
-  if (table_of_sets == nullptr) {
+  if (table_of_sets == nullptr || !set || !type) {
     return (nullptr);
```

Comment:
`return` IS NOT A FUNCTION. Sigh. `return nullptr;`.

Reply:
Don't know why people did that.